spec/features/professors/index_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe "Index" do
+  before :each do
+    @snape = Professor.create(name: "Severus Snape", age: 45, specialty: "Potions")
+    @hagarid = Professor.create(name: "Rubeus Hagrid", age: 38 , specialty: "Care of Magical Creatures")
+    @lupin = Professor.create(name: "Remus Lupin", age: 49 , specialty: "Defense Against The Dark Arts")
+
+    @harry = Student.create!(name: "Harry Potter" , age: 11 , house: "Gryffindor" )
+    @malfoy = Student.create!(name: "Draco Malfoy" , age: 12 , house: "Slytherin" )
+    @longbottom = Student.create!(name: "Neville Longbottom" , age: 11 , house: "Gryffindor" )
+
```

Comment:
This test is only testing information displayed about a professor, so it doesn't need to setup any data related to a student. Make sure that you avoid unnecessary data setup in your tests, at scale this will make your test suite run much slower.